service/static/js/rest_api.js

```diff
@@ -30,6 +30,14 @@ $(function () {
         }
     }
 
+    function update_address_form(res) {
```

Comment:
this throws an error in the browser when res is null, maybe just add a conditional check to make sure that res is not null before trying to access res.address_id